src/lsg/patterns/property-items/boolean-item/index.tsx

```diff
@@ -15,62 +15,77 @@ interface IndicatorProps {
-`;
-
-const indicatorWidth = 42;
-const indicatorHeight = 24;
-const indicatorBorderWidth = 1;
+const indicatorWidth = 48;
+const indicatorHeight = 30;
 
 const StyledIndicator = styled.span`
 	position: relative;
-	display: block;
+	display: inline-block;
 	width: ${indicatorWidth}px;
 	height: ${indicatorHeight}px;
 	border-radius: ${indicatorHeight / 2}px;
-	background: ${colors.grey90.toString()};
 	box-sizing: border-box;
-	box-shadow: inset 0 0 0 ${indicatorBorderWidth}px ${colors.grey60.toString()};
-
-	&:after {
-		content: '';
+	box-shadow: inset 0 0 0 1px ${colors.grey80.toString()};
+	transition: all ease-in-out 0.1s;
+	user-select: none;
+	&::after {
+		position: absolute;
+		content: url('data:image/svg+xml; utf8, <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 12"><path d="M6,6L4.5,7.5L6,6L4.5,4.5L6,6z M6,6l1.5-1.5L6,6l1.5,1.5L6,6z" fill="none" stroke="${colors.grey60.toString()}" stroke-width="1" stroke-linecap="round" stroke-linejoin="round" /></svg>');
 		display: block;
 		width: ${indicatorHeight}px;
 		height: ${indicatorHeight}px;
-		border: ${indicatorBorderWidth}px solid ${colors.grey60.toString()};
-		transform: translateX(0px);
+		transform: translateX(${indicatorWidth - indicatorHeight}px);
 		border-radius: 100%;
 		background: ${colors.white.toString()};
-		transition: all ease-in-out 0.25s;
+		transition: all ease-in-out 0.1s;
 		box-sizing: border-box;
+		box-shadow: inset 0 0 0 1px ${colors.grey60.toString()};
+	}
+	&:hover {
+		&::after {
+			box-shadow: inset 0 0 0 1px ${colors.grey60.toString()}, 0.5px 0.5px 3px ${colors.grey60.toString()};
+		}
 	}
 	${(props: IndicatorProps) =>
 		props.checked
 			? `
-			background: ${colors.blue40.toString()};
-			box-shadow: inset 0 0 0 ${indicatorBorderWidth}px ${colors.blue40.toString()};
-
-			&:after {
-				transform: translateX(${indicatorWidth - indicatorHeight}px);
+			background: ${colors.blue80.toString()};
+			box-shadow: inset 0 0 0 0.5px ${colors.blue40.toString()};
+			&::after {
+				content: url('data:image/svg+xml; utf8, <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 12"><polyline points="3.8,6 5.2,7.5 8.2,4.5 " fill="none" stroke="${colors.blue40.toString()}" stroke-width="1" stroke-linecap="round" stroke-linejoin="round"></polyline></svg>');
+				transform: translateX(0px);
 				background: ${colors.white.toString()};
 				border-color: ${colors.blue40.toString()};
+				box-shadow: inset 0 0 0 1px ${colors.blue40.toString()};
+			}
+			&:hover {
+				&::after {
+					box-shadow: inset 0 0 0 1px ${colors.blue40.toString()}, 0.5px 0.5px 3px ${colors.blue40.toString()};
+				}
 			}
 		`
 			: ''};
 `;
 
 const StyledLabel = styled.span`
-	display: block;
+	display: inline-block;
 	font-size: 12px;
 	font-family: ${fonts().NORMAL_FONT};
-	color: ${colors.grey36.toString()};
-	margin-bottom: ${getSpace(SpaceSize.XS)}px;
+	color: ${colors.grey50.toString()};
+	padding: ${getSpace(SpaceSize.XS) + getSpace(SpaceSize.XXS)}px 0 0;
```

Comment:
Instead if moving the label to center via `padding` we should set `align-items: center` on the the flex container

Reply:
Also a padding to the left and `box-sizing:  border-box` could avoid the missing visual distance here:

```css
box-sizing: border-box;
padding-right: ${getSpace(SpaceSize.XS)};
```
**before**
<img width="144" alt="screen shot 2018-05-08 at 10 28 46" src="https://user-images.githubusercontent.com/4248851/39746863-efd2dc6e-52ab-11e8-943c-72d2c8838527.png">

**after**
<img width="226" alt="screen shot 2018-05-08 at 10 38 43" src="https://user-images.githubusercontent.com/4248851/39746880-ff6e0ca2-52ab-11e8-8469-6fffe6fe5b6b.png">

Reply:
We'l also want to add a `title` attribute to the spans to make the labels readable via a tooltip:

<img width="244" alt="screen shot 2018-05-08 at 10 41 03" src="https://user-images.githubusercontent.com/4248851/39747038-6dfb7380-52ac-11e8-84d9-8ac4363c8eef.png">